.github/workflows/deploy-stage.yml

```diff
@@ -80,6 +80,24 @@ jobs:
           source: ".docker/autoline/docker-compose.stage.yml,.docker/autoline/nginx/nginx.rtmp.conf,.docker/autoline/database/Dockerfile,.docker/autoline/database/user_privileges.sql,.docker/autoline/nginx/nginx.conf"
           target: app
 
+      - name: Copy a file from s3
+        uses: prewk/s3-cp-action@v2
+        with:
+          aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          source: 's3://autoline-secret/secret'
+          dest: '.env/'
+          flags: --recursive
+
+      - name: Copy secret files to server
+        uses: appleboy/scp-action@master
+        with:
+          host: ${{ secrets.SSH_HOST }}
+          username: ${{ secrets.SSH_USERNAME }}
+          key: ${{ secrets.SSH_KEY }}
+          source: .env
+          target: app
```

Comment:
Why don't we copy env files directly to the app directory?

Reply:
We can, but will we have some advances?)